src/core/Microsoft.PowerToys.Settings.UI/Views/PowerRenamePage.xaml

```diff
@@ -89,7 +89,7 @@
                             />
 
             <ToggleSwitch x:Uid="PowerRename_Toggle_RestoreFlagsOnLaunch"
-                          Margin="{StaticResource SmallTopMargin}" 
+                          Margin="0, 17, 0, 0" 
```

Comment:
Will this work with different DPI scaling?

Reply:
Yep, UWP uses effective pixels. I can confirm that the result looks consistent when changing my DPI to e.g. 100%, 150% and 200% (Surface Laptop 3)

https://docs.microsoft.com/en-us/windows/uwp/design/basics/design-and-ui-intro